src/test/regress/input/multi_insert_select_behavioral_analytics_create_table.source

```diff
@@ -41,3 +41,425 @@ CREATE INDEX is_index4 ON events_table(event_type);
 CREATE INDEX is_index5 ON users_table(value_2);
 CREATE INDEX is_index6 ON events_table(value_2);
 
+
+-- Create composite type to use in subquery pushdown
+CREATE TYPE user_composite_type AS
+(
+	tenant_id BIGINT,
+	user_id BIGINT
+);
+
+-- do it on the workers as well
+SELECT run_command_on_workers('
+
+	CREATE TYPE user_composite_type AS
+	(
+		tenant_id BIGINT,
+		user_id BIGINT
+	);
+');
+
+
+CREATE FUNCTION cmp_user_composite_type_function(user_composite_type, user_composite_type) RETURNS int
+LANGUAGE 'internal'
+AS 'btrecordcmp'
+IMMUTABLE
+RETURNS NULL ON NULL INPUT;
+
+-- do it on the workers as well
+SELECT run_command_on_workers('
+
+	CREATE FUNCTION cmp_user_composite_type_function(user_composite_type, user_composite_type) RETURNS int
+	LANGUAGE ''internal''
+	AS ''btrecordcmp''
+	IMMUTABLE
+	RETURNS NULL ON NULL INPUT;
+');
```

Comment:
FWIW, something like `run_command_on_coordinator_and_workers` could also be useful for customers when creating e.g. ENUM types for use in table definitions.